src/main/java/com/bytebreak/animagic/texture/AnimagicTextureRegion.java

```diff
@@ -5,65 +5,47 @@
 import com.badlogic.gdx.math.Vector2;
 
 public class AnimagicTextureRegion extends TextureRegion {
+    public AnimagicTextureData meta = new AnimagicTextureData();
     private Texture normals;
-    private float offsetX = 0;
-    private float offsetY = 0;
 
     public AnimagicTextureRegion(Texture texture, Texture normals) {
         super(texture);
-        init(normals, 0, 0);
+        init(normals);
     }
 
     public AnimagicTextureRegion(Texture texture, Texture normals, int width, int height) {
         super(texture, width, height);
-        init(normals, 0, 0);
+        init(normals);
     }
 
     public AnimagicTextureRegion(Texture texture, Texture normals, int x, int y, int width, int height) {
         super(texture, x, y, width, height);
-        init(normals, 0, 0);
+        init(normals);
     }
 
     public AnimagicTextureRegion(AnimagicTextureRegion region) {
         super(region);
-        init(region.getNormalTexture(), 0, 0);
+        init(region.getNormalTexture());
     }
 
     public AnimagicTextureRegion(TextureRegion region, Texture normals) {
         super(region);
-        init(normals, 0, 0);
+        init(normals);
     }
 
     public AnimagicTextureRegion(AnimagicTextureRegion region, int x, int y, int width, int height) {
         super(region, x, y, width, height);
-        init(region.getNormalTexture(), 0, 0);
+        init(region.getNormalTexture());
     }
 
     public AnimagicTextureRegion(TextureRegion region, Texture normals, int x, int y, int width, int height) {
         super(region, x, y, width, height);
-        init(normals, 0, 0);
+        init(normals);
     }
 
-    public AnimagicTextureRegion(Texture texture, Texture normals, int x, int y, int width, int height, float offsetX, float offsetY) {
```

Comment:
My thoughts here are that we won't need this information any more (the offset parameters). These will be automagically strapped onto the region by the Atlas at the time of loading them. Ideally it will be entirely transparent to us when we are coding.